### ISSUE TITLE
Animation glitch when expanding databound PanelBar with expandMode: single

### ISSUE THREAD

**Issue by Dimitar-Goshev** [telerik/kendo-themes#2990]
**Describe the bug**
Databound PanelBar with expandMode: "single" has two visible nodes at the same time, instead of only one, and during the transition, the nodes jump up and down.

**To reproduce**
1. Run the following [Dojo snippet](https://dojo.telerik.com/@Dimitar-Goshev/eyohUpin)
2. Expand the first node of a PanelBar.
3. Expand the second node of a PanelBar.

**Current behavior**
Both the first and second node are visible and the PanelBar becomes higher than expected. Then the first node is collapsed.

**Expected behavior**
The first node is collapsed while the second node is expanded. The PanelBar height is not changed with a glitch.
Such behavior is reproducible with Kendo UI for jQuery PanelBar LESS themes up to 2021.3.914 included - [Dojo snippet](https://dojo.telerik.com/@Dimitar-Goshev/itaLOVuL). In 2021.3.1109 SASS themes styles were transferred to LESS themes and the issue is reproducible with LESS themes as well.
Such behavior is reproducible with Kendo UI for Angular PanelBar - [StackBlitz snippet](https://stackblitz.com/edit/angular-vxpxyp?file=app/app.component.ts)

**Affected package**
* theme-default
* theme-bootstrap
* theme-material

**Affected suites**
* Kendo UI for jQuery 2021.3.1109

**Affected browsers**
* All

**Workaround** - [Workaround Dojo](https://dojo.telerik.com/@Juveniel/IFoJEliY/2)
```
    <style>
      .k-panelbar .k-panelbar-item { transition: none !important; }
      .k-panelbar .k-link { transition: none !important; }
  </style>
```

**Additional context**
This is not new behavior for Kendo UI for jQuery PanelBar. It is reproducible with [earlier versions](https://docs.telerik.com/kendo-ui/styles-and-layout/sass-version-compatibility) such as Kendo UI 2020.2.513 (R2 2020) and @progress/kendo-theme-bootstrap@4.16.1 , @progress/kendo-theme-default@4.18.1 , @progress/kendo-theme-material@3.15.1

**Comment by epetrow** [comment]
@Dimitar-Goshev This is more likely to be an issue with the Kendo JQuery animations, as it works with the React suite using the same version of the themes: 2021.3.1109. Could you please check with a developer and confirm if this is a bug in the widget? If that is the case, could you please transfer the issue?

**Comment by Dimitar-Goshev** [comment]
@epetrow - kendoAnimate has not been changed recently. The workaround suggests that changing styles fixes the issue and its origin is a change in the styles.

**Comment by silviyaboteva** [comment]
Hey @Dimitar-Goshev,
Researching further the case, I noticed that the transition CSS property is present from day 1 of the PanelBar component and it serves to smoothly transition the color, background-color, border-color and box-shadow of the items. Reference links:
https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/panelbar/_layout.scss#L85 and the variable value: https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/_variables.scss#L293

We cannot remove it as this is essential for the component. 

I noticed something else though. The element `ul.k-panelbar-group` toggles values between these properties:
```css
   display: block; // or hidden;
   height: auto; // or the height of the content in px (expanding) or 0px (collapsing).
   overflow: visible; // hidden only when the animation is running. Then revert back to visible.
```

The issue seems to come from the additional set `transition: all 200ms ease-out 0s` which changes the `display` property to `none` with a delay.

@Juveniel, @TeyaVes may I ask you to check this case and confirm that this is not themes related issue?
